src/test/java/seedu/address/testutil/RawExpenseBuilder.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.testutil;
```

Comment:
Instead of creating a separate class for Building Raw dates, is it possible for you to add a second constructor in ExpenseBuilder.java?

```
    public ExpenseBuilder(boolean isRaw) {
        name = new Name(DEFAULT_NAME);
        amount = new Amount(DEFAULT_AMOUNT);
        date = new Date(DEFAULT_DATE, isRaw);
        tags = new HashSet<>();
    }
```

Reply:
is it a good idea? im not sure

Reply:
The diff of RawExpenseBuilder and ExpenseBuilder is just that boolean, thus the suggestion. 2 Usages of RawExpenseBuilder, are in ExpenseListParser:39 & 60